db/db.js

```diff
@@ -49,4 +49,4 @@ function getAllData() {
   });
 }
 
-module.exports = { getAllData };
+module.exports = { getData };
```

Comment:
change this query name to be match its new functionality of only grabbing some data and not ALL data like it was before refactoring.